packages/components/src/components/interface/EventTopBar.tsx

```diff
@@ -28,15 +28,21 @@ const TopBar = styled.div`
   position: fixed;
   z-index: 1;
 `
-const TopBarTitle = styled.h4`
+const TopBarTitle = styled.h4<{ TopBarActions?: boolean }>`
   ${({ theme }) => theme.fonts.h4};
-  padding-left: 16px;
+  padding-left: ${({ TopBarActions }) => (TopBarActions ? `6px` : `16px`)};
```

Comment:
I don't think it's a good idea to make the padding of the title dependent on the `TopBarActions`